src/pages/index.js

```diff
@@ -43,61 +80,152 @@ const profileEditPopup = new PopupWithForm(
 );
 profileEditPopup.setEventListeners();
 
-/* --------------------------- Popup Preview Image -------------------------- */
+/* --------------------------- Preview Image Popup -------------------------- */
 
 const imagePopUp = new PopUpWithImage("#preview-image-popup");
 imagePopUp.setEventListeners();
 
-/* --------------------------- Popup Add Card -------------------------- */
+/* -------------------------- Add Card Popup ------------------------------- */
 
 const addCardPopUp = new PopupWithForm(
   "#add-new-card-popup",
   handleAddCardFormSubmit
 );
 addCardPopUp.setEventListeners();
 
-/* -------------------------------------------------------------------------- */
-/*                                   Section                                  */
-/* -------------------------------------------------------------------------- */
+/* ---------------------------Confirm Delete Popup-------------------------- */
 
-const cardSection = new Section(
-  {
-    items: initialCards,
-    renderer: (item) => {
-      console.log(item);
-      const cardEl = renderCard(item);
-      cardSection.addItem(cardEl);
-    },
-  },
-  ".cards__list"
+const confirmDeletePopup = new PopupWithConfirmation(
+  "#confirm-image-delete-popup",
+  handleDeleteConfirmSubmit
 );
-cardSection.renderItems();
+confirmDeletePopup.setEventListeners();
 
-const userInfoNew = new UserInfo(".profile__name", ".profile__description");
+/* ------------------------Change Profile Avatar Popup---------------------- */
+
+const changeProfileAvatarPopUp = new PopupWithForm(
+  "#change-profile-avatar-popup",
+  handleChangeProfileAvatarFormSubmit
+);
+changeProfileAvatarPopUp.setEventListeners();
+
+const userInfoNew = new UserInfo(
+  profileName,
+  profileDescription,
+  profileAvatar
+);
+
+api
+  .fetchUserInfo()
+  .then((data) => (userData = data))
```

Comment:
Needs correction: Please remove this `then` block, it does nothing

Your code should look like the following:

```js
api
  .fetchUserInfo()
  .then((userData) => {
    userInfoNew.setUserInfo(userData.name, userData.about);
    userInfoNew.setUserAvatar(userData.avatar);
  })
```